neqo-http3/src/connection_client.rs

```diff
@@ -357,10 +358,11 @@ impl Http3Client {
             stream.priority_update(priority);
             self.base_handler
                 .control_stream_local
```

Comment:
please also do not expose control_stream_local.
Create a function queue_update_priority in Http3Connection that calls `self.control_stream_local.queue_update_priority().`